app/controllers/users_controller.rb

```diff
@@ -11,13 +12,20 @@ def show; end
 
   def new
     @user = User.new
+    @user.build_profile
+    @user.profile.build_contact
```

Comment:
You can also do something similar as @casaper did in #100 to move this logic out of the controller.